### ISSUE TITLE
Serve to local IP address?

### ISSUE THREAD

**Issue by lazarljubenovic** [lukejacksonn/servor#18]
`create-react-app` automatically serves to localhost and a local IP address so I can easily access the website from my phone. I have literally 0 idea on how this works or how difficult it is to do, but would love to see in servor!

**Comment by lukejacksonn** [comment]
This works already! So long as your machine and phone are on the same network you can just swap out `localhost` for your local IP address. You can find your local IP on a mac by doing something like:

```
ifconfig | grep "inet " | grep -Fv 127.0.0.1 | awk '{print $2}'
```

Then on your phone just visit that address with the port your server is running on, something like:

```
192.168.1.XXX:8080
```

Servor could run this command under the hood and console log it out on startup. That would be trivial enough to implement. Would review a PR for that is you are interested.

**Comment by lazarljubenovic** [comment]
Oh, it does work! Thanks a lot. :)

Certainly, writing it in console would help greatly. I don't have access to Windows or Mac (Ubuntu only), so it's better if someone who can test it on multiple platforms fiddles with that.

**Comment by lukejacksonn** [comment]
Implemented in #27 🎉 thanks for the suggestion @lazarljubenovic
